contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+     *  @param _parties The `parties` value of the agreement.
+     *  @param _numberOfChoices The `numberOfChoices` value of the agreement.
+     *  @param _extraData The `extraData` value of the agreement.
+     *  @param _arbitrationFeesWaitingTime The `arbitrationFeesWaitingTime` value of the agreement.
+     *  @param _arbitrator The `arbitrator` value of the agreement.
+     */
+    function _createAgreement(
+        bytes32 _tokenID,
+        string _metaEvidence,
+        address[] _parties,
+        uint _numberOfChoices,
+        bytes _extraData,
+        uint _arbitrationFeesWaitingTime,
+        Arbitrator _arbitrator
+    ) internal {
+        itemIDToAgreementCount[_tokenID]++;
+        bytes32 agreementID = keccak256(abi.encodePacked(_tokenID, itemIDToAgreementCount[_tokenID]));
+        agreementIDtoItemID[agreementID] = _tokenID;
+
+        super._createAgreement(
+            agreementID,
+            _metaEvidence,
+            _parties,
+            _numberOfChoices,
+            _extraData,
+            _arbitrationFeesWaitingTime,
+            _arbitrator
+        );
+    }
+
+    /** @dev Executes the ruling on the specified agreement.
+     *  @param _agreementID The ID of the agreement.
+     *  @param _ruling The ruling.
+     */
+    function executeAgreementRuling(bytes32 _agreementID, uint _ruling) internal {
+        super.executeAgreementRuling(_agreementID, _ruling);
+        uint256 disputeID = agreements[_agreementID].disputeID;
+        Item storage item = items[disputeIDToItem[disputeID]];
+        Agreement storage agreement = agreements[_agreementID];
+        require(agreement.disputed, "The item is not disputed.");
+
+        if (_ruling == REGISTER) {
+            if (item.status == ItemStatus.Resubmitted || item.status == ItemStatus.Submitted)
+                agreement.parties[0].send(item.balance); // Deliberate use of send in order to not block the contract in case of reverting fallback.
+            else
+                agreement.parties[1].send(item.balance);
+
+            item.status = ItemStatus.Registered;
+        } else if (_ruling == CLEAR) {
+            if (item.status == ItemStatus.PreventiveClearingRequested || item.status == ItemStatus.ClearingRequested)
+                agreement.parties[0].send(item.balance);
+            else
+                agreement.parties[1].send(item.balance);
+
+            item.status = ItemStatus.Cleared;
+        } else { // Split the balance 50-50 and give the item the initial status.
+            if (item.status == ItemStatus.Resubmitted)
+                item.status = ItemStatus.Cleared;
+            else if (item.status == ItemStatus.ClearingRequested)
+                item.status = ItemStatus.Registered;
+            else
+                item.status = ItemStatus.Absent;
+
+            agreement.parties[0].send(item.balance / 2);
+            agreement.parties[1].send(item.balance / 2);
+        }
+
+        agreement.disputed = false;
```

Comment:
Why do this if we are not reusing agreements. This might interfere with other logic.

Reply:
I'm not sure I understand. We are reusing agreements.